### DISCUSSION TITLE
Nightly Build Snapshots Problem

### DISCUSSION THREAD

**Discussion by ghost** [OpenCCU/OpenCCU#2264] [Development]
CCU3 verhält sich nach der Installation von RaspberryMatic-3.69.2.20230315-f6177c-rpi3 seltsam. Das Problem ist auch in dieser Version von RaspberryMatic-3.69.3.20230317-3eec3e-rpi3 vorhanden. 
Es dreht sich alles um die rote LED, die anzeigt, wenn das Gerät ausgeschaltet ist. Nach der Installation der neuen Version blinkt die rote LED die ganze Zeit ( Es geht nicht um diese kleine rote LED, die die Aufnahme signalisiert )

**Comment by ghost** [comment]
...

**Reply by ghost** [reply]
das Gerät verhält sich wie ausgeschaltet (obwohl es normal funktioniert)

**Reply by jens-maus** [reply]
Bitte beschreib das nochmal genauer: Wie ist es normal (d.h. bei der aktuellen stable version) und wie ist es bei der aktuellen nightly snapshot. Du hast das hier ein wenig durcheinander beschrieben und aufgezeigt.

**Reply by ghost** [reply]
Entschuldigung für meine deutsche Sprache.
Die rote LED sollte nicht normal leuchten. In der stabilen Version leuchten keine Lichter. Was ich im Video gezeigt habe, sollte nur passieren, wenn das Gerät ausgeschaltet ist ( shutdown)


Wahrscheinlich wird es später jemand bestätigen

**Reply by jens-maus** [reply]
> Entschuldigung für meine deutsche Sprache.

Then use english if you are not confident enough in explaining the issue.

> Die rote LED sollte nicht normal leuchten. In der stabilen Version leuchten keine Lichter. Was ich im Video gezeigt habe, sollte nur passieren, wenn das Gerät ausgeschaltet ist ( shutdown)

What do you mean with "normal leuchten"? In the video it blinks and does not show a constant red light. So please again: explain it in more detail: How is it in the current stable version, how is it in the nightly build and what do you expect.

**Reply by ghost** [reply]
Normally, when the device starts and working, I don't see any red light - this is how it looks in the stable version ( because I turned them off with one program ), after installing the new version, the red light is on , similar to when the device is turned off, but it flashes. 
Ps. when writing data to the card, this light must burns very dimly  ( stable version ) , and after installing the new version the light is VERY bright and flashes periodically, even after a few hours

**Comment by ghost** [comment]
this is how it looks on the stable version RaspberryMatic-3.67.10.20230225-rpi3 : 

https://user-images.githubusercontent.com/81189856/226083647-924653e1-85dc-40c5-a0eb-be5e275339cc.mp4


.
.
.
.



and here new Nightly Build RaspberryMatic-3.69.3.20230318-4ebec1-rpi3: 


https://user-images.githubusercontent.com/81189856/226083884-098b89b9-7de1-41b5-8af7-95c89e947fcb.mp4

**Comment by Baxxy13** [comment]
Da hat sich wohl was bezüglich der Raspi-Leds verändert.
Aus Led0 / Led1 wurde:
```
root@RM-PI4B-2GB-45-Test:~# ls -al /sys/class/leds
total 0
drwxr-xr-x    2 root     root             0 Jan  1  1970 .
drwxr-xr-x   60 root     root             0 Jan  1  1970 ..
lrwxrwxrwx    1 root     root             0 Mar 18 09:47 ACT -> ../../devices/platform/leds/leds/ACT
lrwxrwxrwx    1 root     root             0 Mar 18 09:47 PWR -> ../../devices/platform/leds/leds/PWR
```

**Reply by jens-maus** [reply]
Danke, das hab ich gerade auch festgestellt gehabt und nun in https://github.com/jens-maus/RaspberryMatic/commit/1dfe9bb3f5d1e4d715ee774a5c9c70250a5e60b7 entsprechend angepasst/repariert.

**Comment by ghost** [comment]
I am glad that the problem has been solved and finally correctly diagnosed ;-) Thanks Jens and Baxxy
